content/faq.md

```diff
@@ -7,45 +7,64 @@ navbar = "page"
 
 # Frequently Asked Questions
 
+### What's different this year?
+
+Everything! For one, MinneHack will be held online due to the current world event. The topics and
```

Comment:
maybe "pandemic" instead of "world event"? when it said event the first thing i thought of was minnehack

Reply:
ok